package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Bucklescript PPX which generates JSON (de)serializers for user-defined types",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest lib/js/test",
```

Comment:
Updated the test script since jest tries to run tests inside node_modules or any _opam/_esy file.